### PR TITLE
Adding a feature for Aggregation that, for now, tests min and max

### PR DESCRIPTION
**Pull request by sherfert**
This contains several test for min and max with different types.

### CODE REVIEW

File: tck/features/Aggregation.feature [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+Feature: Aggregation
+
+  Scenario: `max()` should aggregate strings
```
[the remark this answers is not in the document]
Comment (Mats-SX) [reply tck/features/Aggregation.feature]: Same comment for names below.

File: tck/features/Aggregation.feature [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+    Then the result should be:
+      | min(x) |
+      | 0.1    |
+    And no side effects
+
+  Scenario: `max()` should aggregate mixed values
+    Given any graph
+    When executing query:
+      """
+      UNWIND [1, 'a', null, [1,2], 0.2, 'b'] AS x
```
Comment (Mats-SX) [inline tck/features/Aggregation.feature]: style: space after comma in inner list

File: tck/features/Aggregation.feature [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+    Then the result should be:
+      | max(x) |
+      | 1      |
+    And no side effects
+
+  Scenario: `min()` should aggregate mixed values
+    Given any graph
+    When executing query:
+      """
+      UNWIND [1, 'a', null, [1,2], 0.2, 'b'] AS x
```
Comment (Mats-SX) [inline tck/features/Aggregation.feature]: style: space after comma in inner list

File: tck/features/Aggregation.feature [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+      | min(x) |
+      | [1,2]  |
+    And no side effects
+
+  Scenario: `max()` should aggregate list values
+    Given any graph
+    When executing query:
+      """
+      UNWIND [[1], [2], [2, 1]] AS x RETURN
+      max(x)
```
Comment (Mats-SX) [inline tck/features/Aggregation.feature]: style: weird newline

File: tck/features/Aggregation.feature [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+      | max(x) |
+      | [2, 1] |
+    And no side effects
+
+  Scenario: `min()` should aggregate list values
+    Given any graph
+    When executing query:
+      """
+      UNWIND [[1], [2], [2, 1]] AS x RETURN
+      min(x)
```
Comment (Mats-SX) [inline tck/features/Aggregation.feature]: style: weird newline

File: tck/features/Aggregation.feature [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+      """
+    Then the result should be:
+      | min(x) |
+      | [1] |
```
Comment (Mats-SX) [inline tck/features/Aggregation.feature]: Misaligned result table

File: tck/features/Aggregation.feature [hunk 7]
```diff
@@ -0,0 +1,161 @@
...
+  Scenario: `min()` should aggregate mixed values
+    Given any graph
+    When executing query:
+      """
+      UNWIND [1, 'a', null, [1,2], 0.2, 'b'] AS x
+      RETURN min(x)
+      """
+    Then the result should be:
+      | min(x) |
+      | [1,2]  |
```
Comment (Mats-SX) [inline tck/features/Aggregation.feature]: style: space after comma

**Comment by petraselmer** [comment]
I really like this a lot. Very comprehensive.

**Pull request merged**